### DISCUSSION TITLE
make performancePointCounter able to show min/max achievable value

### DISCUSSION THREAD

**Discussion by huaji2369** [ppy/osu#24945] [Ideas]
let performancePointCounter able to show in minimum/maximum achievable mode like the accuracyCounter did

**Comment by Rekunan** [comment]
The modes in `accuracyCounter` changes how it calculates accuracy, to have something similar in `performancePointCounter` it'd most likely have to be as a fraction such as: `current pp`/`max pp`.

**Reply by Natelytle** [reply]
no? thats not what he wants, he wants the numerical value for if you SSd the rest of the map vs if you missed the rest of the map which is trivial to do

**Reply by Rekunan** [reply]
Ohhh that calculation. My suggestion is also trivial, so I could see all being available.

**Reply by huaji2369** [reply]
maybe max achievable only, min achievable for this doesn't make much sense
